pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -105,6 +105,7 @@ stdenv.mkDerivation rec {
       ))
       (lib.withFeature ncursesSupport "ncursesw")
       (lib.withFeature systemdSupport "systemd")
+      (lib.withFeatureAs systemdSupport "sysusersdir" "${placeholder "bin"}/lib/sysusers.d")
```

Comment:
Is bin the right output for this?

Reply:
https://www.freedesktop.org/software/systemd/man/latest/tmpfiles.d.html

https://www.freedesktop.org/software/systemd/man/latest/sysusers.d.html

I'd guess out is correct, considering that's where we put tmpfiles.d.